terraform/es.tf

```diff
@@ -0,0 +1,35 @@
+resource "aws_elasticsearch_domain" "monitoring-framework" {
+  domain_name           = "tg-${var.environment}-es"
+  elasticsearch_version = "2.3"
+
+  cluster_config {
+    instance_type            = "m4.large.elasticsearch"
+    instance_count           = 1
+    dedicated_master_enabled = false
+    dedicated_master_type    = "m4.large.elasticsearch"
+    dedicated_master_count   = 1
+  }
+
+  ebs_options {
+    ebs_enabled = true
+    volume_size = 50
+  }
+
+}
+
+resource "aws_elasticsearch_domain_policy" "monitoring-framework-policy" {
+  domain_name = aws_elasticsearch_domain.monitoring-framework.domain_name
+  access_policies = <<POLICIES
```

Comment:
@avivgu Please change to aws_iam_policy_document

Reply:
done